component/faro/receiver/internal/payload/utils.go

```diff
@@ -0,0 +1,73 @@
+package payload
+
+import (
+	"fmt"
+	"sort"
+
+	om "github.com/wk8/go-ordered-map"
+)
+
+// KeyVal is an ordered map of string to interface
+type KeyVal = om.OrderedMap
```

Comment:
Will this be a problem if we define two separate faro.receiver components?

Reply:
Why would it be? This is a type alias, not a variable :)

Reply:
Oops, sorry I needed my second coffee of the day 😅